webrender/examples/animation.rs

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+include!("boilerplate.rs");
```

Comment:
this is extremely useful to have the boilerplate isolated from the examples, but I think there is a better way to share it. See this link (with a `common` folder) for an example - https://github.com/brendanzab/cgmath/tree/master/benches